video/out/vo_gpu_next.c

```diff
@@ -1695,7 +1695,7 @@ const struct m_opt_choice_alternatives lut_types[] = {
 const struct vo_driver video_out_gpu_next = {
     .description = "Video output based on libplacebo",
     .name = "gpu-next",
-    .caps = VO_CAP_ROTATE90,
+    .caps = VO_CAP_ROTATE90 | (defined(PL_HAVE_LAV_FILM_GRAIN) * VO_CAP_FILM_GRAIN),
```

Comment:
It looks like you're trying to use a pre-processor directive `#if defined(foo)` but this doesn't look like it's going to work since there's no `#if` here.